src/microbe_stage/PatchMap.cs

```diff
@@ -219,13 +223,19 @@ public void RemoveExtinctSpecies(bool playerCantGoExtinct = false)
                 entry.Value.RemoveSpecies(item.Key);
                 GD.Print("Species ", item.Key.FormattedName, " has gone extinct in ",
                     entry.Value.Name);
+                result.Add(item.Key);
```

Comment:
Isn't this logic wrong? This adds species that don't have population in *this* patch to the list of extinct species.

Reply:
Is this PR old enough that I can pull it and apply this fix correctly? @hhyyrylainen

Reply:
I think it's been long enough that this PR can be considered stuck and can be picked up by someone else.
I already put it in the stuck column on the planning board.